server/src/tasks/schema.js

```diff
@@ -1,5 +1,5 @@
 const { pubSub } = require('../subscriptions')
-const { conflictHandler } = require("@aerogear/voyager-conflicts")
+const { conflictHandler } = require("offix-conflicts-server")
```

Comment:
Waiting for the release of the voyager-server
CC: @darahayes

Reply:
Working on that today

Reply:
Right. Let me know once you have any release. I want to make sure we are testing the right package.